.travis.yml

```diff
@@ -27,3 +27,8 @@ script:
   - make test_ftp
   - make test_bioblend
   - make test_docker_in_docker
+
+# blocklist
+branches:
```

Comment:
This is probably better: https://docs.travis-ci.com/user/customizing-the-build#Skipping-a-build